internal/server/add_complaint.go

```diff
@@ -107,8 +109,11 @@ type addComplaintData struct {
 	Success   bool
 	Error     sirius.ValidationError
 
-	Categories map[string]complaintCategory
-	Complaint  sirius.Complaint
+	Categories            map[string]complaintCategory
```

Comment:
Not required for this ticket but im curious as to why we hardcode the categories and dont fetch them from the API instead :raised_eyebrow:

Reply:
They're not actually stored in the API, only the frontend! If you manipulate the request, there's nothing stopping you setting the category/subcategory to any string you like. [VEGA-1667](https://opgtransform.atlassian.net/browse/VEGA-1667) should address this.